apps/dashboard/features/holders-and-delegates/components/Delegates.tsx

```diff
@@ -6,17 +6,18 @@ import {
   HoldersAndDelegatesDrawer,
 } from "@/features/holders-and-delegates";
 import { TimeInterval } from "@/shared/types/enums";
-import { TheTable, SkeletonRow } from "@/shared/components";
+import { TheTable, SkeletonRow, BlankState } from "@/shared/components";
```

Comment:
This component's name is BlankSlate, but I think this is not a change for this issue